health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_dcgm_watcher/test_dcgm.py

```
@@ -1234,3 +1234,57 @@ def test_other_nvlink_error_code_not_suppressed(self, tmp_path: Path) -> None:
+            release_workers.set()
+
+        dcgm_handle_mock.Shutdown.side_effect = cleanup
+        mock_dcgm_handle.return_value = dcgm_handle_mock
+
+        dcgm_group_mock = MagicMock()
+        # A timeout from the health check is what flags connectivity failure.
+        dcgm_group_mock.health.Check.side_effect = dcgm_structs.DCGMError_Timeout()
+        mock_dcgm_group.return_value = dcgm_group_mock
+
+        # The first cycle only connects; the health check runs on the second.
+        exit = MagicMock(spec=Event)
```

Ruff (0.16.1)

[error] Variable `exit` is shadowing a Python builtin

(A001)